pages/templates/pages/group_index_page.html

```diff
@@ -42,6 +35,10 @@ <h4 class="mb20 mt0">No group in your area?</h4>
   </div>
 </div>
 
+<div class="component pa20 visible-xs f5f5f5-bg">
```

Comment:
@cjmabry actually one issue is I suggest we be more consistent about how we use the bootstrap grid pattern. Generally should have container/row/col pattern for layout, right? I don't understand what a component is.

Reply:
Suggest to follow docs here http://getbootstrap.com/docs/3.3/css/#grid
```
Rows must be placed within a .container (fixed-width) or .container-fluid (full-width) for proper alignment and padding.
Use rows to create horizontal groups of columns.
Content should be placed within columns, and only columns may be immediate children of rows.
...
```